dataprofiler/tests/reports/test_graphs.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+import warnings
+
+from dataprofiler.profilers import IntColumn
+from dataprofiler.reports import graphs
+from unittest import mock
+import dataprofiler as dp
+import pandas as pd
+
+
+@mock.patch("dataprofiler.reports.graphs.plt.show")
+@mock.patch("dataprofiler.reports.graphs.sns")
+class TestPlotHistograms(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.data = [[1, 1.0, 'a', '1/2/2021'],
+                    [None, None, 'b', '1/2/2020'],
+                    [3, 3.5, 'c', '1/2/2022'],
+                    [4, 4.5, 'd', '1/2/2023'],
+                    [5, 6.0, 'e', '5/2/2020'],
+                    [None, None, 'f', '1/5/2020'],
+                    [1, 1.0, 'g', '2/5/2020'],
+                    [None, 10.0, 1, '3/5/2020']]
+        cls.options = dp.ProfilerOptions()
+        cls.options.set({"data_labeler.is_enabled": False})
+        cls.profiler = dp.StructuredProfiler(cls.data, options=cls.options)
+
+    @mock.patch("dataprofiler.reports.graphs.plot_col_histogram")
+    def test_no_columns_specified(self, plot_col_mock, seaborn_mock, plt_mock):
+        graphsplot = graphs.plot_histograms(self.profiler)
+        self.assertEqual(2, plot_col_mock.call_count)
+        # grabs the first argument passed into the plot col call and validates
+        # it is the column profiler and its name matches what we expect it to
+        self.assertEqual(0, plot_col_mock.call_args_list[0][0][0].name)
+        # grabs the second argument passed into the plot col call and validates
+        # it is the column profiler and its name matches what we expect it to
+        self.assertEqual(1, plot_col_mock.call_args_list[1][0][0].name)
+        self.assertIsNotNone(graphsplot)
+
+    @mock.patch("dataprofiler.reports.graphs.plot_col_histogram")
```

Comment:
we can prob move this to above the class since we can apply to all the functions here